homeassistant/components/esphome/assist_satellite.py

```diff
@@ -344,6 +368,11 @@ def handle_timer_event(
             timer_info.is_active,
         )
 
+    async def handle_announce_finished(self, media_id: str) -> None:
```

Comment:
Why is this async

Reply:
Matching the other handle_* callbacks for voice. Inside `aioesphomeapi` these get scheduled as tasks within a callback.